client/client.go

```diff
@@ -963,3 +984,93 @@ func (r *NotaryRepository) DeleteTrustData() error {
 	}
 	return nil
 }
+
+// ListRootCerts returns current trusted root certificates.
+// (In particular, not old certificates which we have already
+// rotated from but are still using for signing to allow rollover,
+// even if they are still trusted locally.)
+func (r *NotaryRepository) ListRootCerts() ([]*x509.Certificate, error) {
+	_, err := r.Update(false)
+	if err != nil {
+		return nil, err
+	}
+
+	keyIDs := r.tufRepo.Root.Signed.Roles[data.CanonicalRootRole].KeyIDs
```

Comment:
@endophage I think we still have cert pinning in place in that PR for TOFU, but yes that makes sense.